setup.py

```diff
@@ -74,6 +74,7 @@
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```

Comment:
Currently there is no PySCF for 3.9. Should the optional install line be modified.

torch has 3.9 pkgs and now includes both 3.8 and 3.9 ones for win32 so potentially that optional line can be updated

Also for CPLEX I see the latest version supports 3.7 and 3.8 - so its added 3.8 support and dropped 3.6. This is not related directly to the 3.9 change but I was checking the optional pkgs against 3.9

Reply:
I see wheels for pyscf on python 3.9. https://pypi.org/project/pyscf/1.7.5.1/#files has cp39 manylinux wheels and https://pypi.org/project/pyscf/1.7.5.2/#files has macOS wheels, so I think we're good there.

For torch I'll remove the version cap on non-linux (it'll just end up becoming != win32).

For cplex that's fine, pip is smart enough to try older versions if the user is on py36 (assuming there is a package that matches the version constraints on the package and is installable on py36)

Reply:
Actually torch has windows amd64 wheels now too: https://pypi.org/project/torch/#files so I removed all the system constraints on it. Anyway everything is updated in: https://github.com/Qiskit/qiskit-aqua/pull/1504/commits/1a2e3496896e314aebf94b641bc20b1304873129

Reply:
I am not sure with PySCF how I missed seeing the wheels there yesterday - I even looked at very latest releases that were there, which looking again I see do have both mac and linux.